### PR TITLE
Draft: Install documentation language reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: userguide/install.rst [hunk 1]
```diff
@@ -146,18 +156,19 @@ Meizu devices are pretty much stuck on Flyme. While the MX4 can be flashed succe
 
 .. warning::
 
-    If you select DOWNLOAD ONLY rather than FIRMWARE UPGRADE, you will end up with a useless brick rather than a fancy Ubuntu Touch device. Be sure to select FIRMWARE UPGRADE.
+    You must select FIRMWARE UPGRADE, and not DOWNLOAD ONLY. Otherwise you brick your device.
 
-    Also, after selecting the ``*Android_scatter.txt``, the drop-down menu choice is reset to DOWNLOAD ONLY. Be sure to select it after selecting the file.
+    Also, after selecting the ``*Android_scatter.txt``, the drop-down menu choice is reset to DOWNLOAD ONLY.
+    Be sure to select it after selecting the file.
```
Comment (comradekingu) [inline userguide/install.rst:162]:
```suggestion
    Make sure it is set to FIRMWARE UPGRADE after selecting the file.
```
?

**Comment by doniks** [comment]
@comradekingu did you notice that this PR failed to build? Actually, all of your open PR's are failing right now. Two of them with a weird git error that I don't understand, but the others with genuine problems in your changes. Please check.

About this PR 399 specifically, It seems you are proposing some changes, but you are also opening a question on it yourself (I think also on another PR). I, personally cannot answer it myself. What should we do? If you want to still gather feedback and work on this, then maybe mark it as draft. Otherwise, maybe remove the particular passage that you are unsure about (for now) and we merge without ...

**Comment by comradekingu** [comment]
@doniks I really don't want to guess which one of two options bricks user devices. From my point of view the distinction for which one to use isn't clear at all. That is the one thing to fix, if anything. Will mark as draft.

**Comment by doniks** [comment]
I see. Maybe some ppl here https://github.com/ubports/docs.ubports.com/issues/388 have used this tool and can answer

**Comment by doniks** [comment]
So .... what do we do? It seems that so far no answer has emerged. I agree that we shouldn't light heartedly publish statements that can brick peoples devices. 

My suggestion would be to simply leave the questionable sections unchanged. In that way we are not making matters worse on that note, but we would improve the other parts of the text with your contribution

**Comment by doniks** [comment]
seems to me this is stalled, so I'm closing. feel free to reopen if you intend to proceed!

**Pull request closed without merging**